107-awesome-strimzi-repo.md

```diff
@@ -0,0 +1,88 @@
+# Add `awesome-strimzi` repository with Strimzi community-created resources
+
+This proposal recommends creating a new GitHub repository, `awesome-strimzi`, under the Strimzi organization.
+Its purpose is to maintain a curated list of community-created resources related to Strimzi, such as blog posts, articles, libraries, frameworks, tools, and other useful materials that are not produced by official Strimzi contributors.
+
+Examples include:
+
+* libraries or SDKs that simplify interaction with Strimzi.
+* blog posts, tutorials, or case studies highlighting Strimzi usage.
+* community tools and integrations.
+
+## Current situation
+
+Currently, there is no dedicated place within the Strimzi project to showcase community-created content.
+
+When users want to share articles or blog posts on their personal or company websites, or even tools or libraries related to Strimzi, there is no central location to collect and share these links.
+This makes it harder for new and existing users to discover the broader ecosystem of community contributions around Strimzi.
+
+## Motivation
+
+Strimzi benefits from a vibrant community whose work extends far beyond the official project deliverables.
+Having a central, community-curated list will:
+
+* showcase the breadth of the Strimzi ecosystem.
+* help users discover useful external resources.
+* encourage community contributions and collaboration.
+* amplify the impact of Strimzi beyond the official channels.
+
+## Proposal
+
+This proposal is about creating a new `awesome-strimzi` repository on GitHub under the Strimzi organization where several external resources related to Strimzi could be published.
+This approach seems to be commonly used for several subjects across the Internet, taking as examples the [awesome-amqp](https://github.com/xinchen10/awesome-amqp) and [awesome-kafka](https://github.com/semantalytics/awesome-kafka).
+They are not official ones while we would like to have an official one within the Strimzi project to make it clear that such resources are all "approved" by the Strimzi contributors.
+
+Create a new GitHub repository named `awesome-strimzi` within the Strimzi organization.
+
+This repository will:
+
+* follow the popular awesome list format used across other projects.
+* contain curated, categorized links to external Strimzi-related resources.
+* be maintained by Strimzi contributors, ensuring the content is relevant, appropriate, and of good quality.
+
+Similar initiatives, but not official, are [awesome-amqp](https://github.com/xinchen10/awesome-amqp) and [awesome-kafka](https://github.com/semantalytics/awesome-kafka).
+Unlike unofficial community lists, this will be an official Strimzi-hosted list to give added visibility and credibility to community contributions.
```

Comment:
Should the previous (or this) lines be removed? As it seems duplicated?

Reply:
Yes, but you are mentioning the repos two times, so I was thinking if it would make sense to rewrite it a bit, so both lines will be merged into one - mentioning that those are not official, but Strimzi's will be.

Reply:
I merged them.